app/controllers/instructor/lessons_controller.rb

```diff
@@ -30,6 +30,12 @@ def require_authorized_for_current_lesson
     end
   end
 
+  def require_authorized_for_current_section
```

Comment:
This method was defined above as a `before_action` yet was never defined, causing error messages when we try to add them.